### PR TITLE
[Manta-PC]Add pallet-collator-selection, align manta-pc'deps with the statemint's

### PR DESCRIPTION
**Pull request by Dengjianping**
1. Add `pallet-collator-selection` to fix getting stuck while authoring blocks.
2. Align manta-pc'deps with the statemint's.
Related commits:
polkadot: 91626111136243164792067392b110d84a129145
substrate: 87e63fa7c70be29383cb95ff52d71874f1db908a
cumulus: 540e74d33c09f27253f0f1c0671315fc035c371f

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -179,11 +186,27 @@ fn testnet_genesis(
...
+			..Default::default()
+		},
+		pallet_session: manta_pc_runtime::SessionConfig {
+			keys: invulnerables
+				.iter()
+				.cloned()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),                 // account id
+						acc.clone(),                 // validator id
```
Comment (ghzlatarev) [inline node/src/chain_spec.rs]: Guess validator and account are the same thing here, right ?

**Comment by ghzlatarev** [comment]
> 1. Add `pallet-collator-selection` to fix getting stuck while authoring blocks.
> 2. Align manta-pc'deps with the statemint's.
>    Related commits:
>    polkadot: 91626111136243164792067392b110d84a129145
>    substrate: 87e63fa7c70be29383cb95ff52d71874f1db908a
>    cumulus: 540e74d33c09f27253f0f1c0671315fc035c371f

Do we have an issue to link  to the PR ?

**Review by ghzlatarev — APPROVED** [review]

File: node/src/chain_spec.rs (hunk 1 above)
Comment (Dengjianping) [reply node/src/chain_spec.rs]: Yes, but we can configure it in the future.

**Comment by ghzlatarev** [comment]
The checks will be fixed with https://github.com/Manta-Network/Manta/pull/79

**Pull request merged**